src/imoh/feature.js

```diff
@@ -0,0 +1,57 @@
+console.log("imoh")
+
+/*
+14 ---------- numbers
+-80 --------- numbers
+8.5 --------- numbers
+true -------- booleans
+(5 < 17) ---- booleans
+null -------- undefined
+3.142 ------- numbers
+"pi" -------- strings
+67+18 ------- function
+*/
+
+
+
+//convert number to base 2
+var base2converter = function(number) {
+	var i = parseInt(number, 10).toString(2);
+	return(i)
```

Comment:
I believe you forgot a semi-colon.

While semi-colons are not necessary, it is highly recommended that you add them at the end of expressions as a missing semi-colon might introduce a bug.